connect/runtime/src/main/java/org/apache/kafka/connect/runtime/standalone/StandaloneConfig.java

```diff
@@ -25,8 +25,18 @@
 public class StandaloneConfig extends WorkerConfig {
     private static final ConfigDef CONFIG;
 
+    /**
+     * <code>offset.storage.file.filename</code>
+     */
+    public static final String OFFSET_STORAGE_FILE_FILENAME_CONFIG = "offset.storage.file.filename";
+    private static final String OFFSET_STORAGE_FILE_FILENAME_DOC = "filename used by FileOffsetBackingStore to save data";
```

Comment:
Here's the other place we should update the config documentation to avoid internal names. Something like `File to store offset data in.` is probably good.